Maquetacion/index.html

```diff
@@ -10,12 +10,82 @@
-    <div class="tech">
-      <img class="logo" src="assets/react_icon.svg" alt="React">
-      <div>
-        <h2 class="text title-card">ReactJS</h2>
-        <p class="text age">Release: 2013</p>
-        <p class="text story">React is a delarative, component based JavaScript library for bouilding user interfaces</p>
+    <div class="section">
+      <div class="tech">
+        <img class="logo" src="assets/react_icon.svg" alt="React">
+        <div>
+          <h2 class="text title-card">ReactJS</h2>
+          <p class="text age">Release: 2013</p>
+          <p class="text story">React is a delarative, component based JavaScript library for bouilding user interfaces</p>
+        </div>
+      </div>
+      <div class="tech">
+        <img class="logo" src="assets/angular_icon.svg" alt="Angular">
+        <div>
+          <h2 class="text title-card">AngularJS</h2>
+          <p class="text age">Release: 2010</p>
+          <p class="text story">AngularJS (commonly reffered to as AngularJS 1.X) is a JavaScript framework for building SPAs with an MVC architecture.</p>
+        </div>
+      </div>
+      <div class="tech">
+        <img class="logo" src="assets/vue_icon.svg" alt="VueJS">
+        <div>
+          <h2 class="text title-card">VueJS</h2>
+          <p class="text age">Release: 2014</p>
+          <p class="text story">Vue.js (pronounced /vjuː/, like view) is an open-source JavaScript framework for building user interfaces.</p>
+        </div>
+      </div>
+      <div class="tech">
+        <img class="logo" src="assets/elm_icon.svg" alt="Elm">
+        <div>
+          <h2 class="text title-card">Elm</h2>
+          <p class="text age">Release: 2016</p>
+          <p class="text story">Elm is a domain-specific programming language for declaratively creating web browser-based graphical user interfaces.</p>
+        </div>
+      </div>
+      <div class="tech">
+        <img class="logo" src="assets/polymer_icon.svg" alt="Polymer">
+        <div>
+          <h2 class="text title-card">Polymer</h2>
+          <p class="text age">Release: 2015</p>
+          <p class="text story">Polymer is an open-source JavaScript library for building web applications using Web Components.</p>
+        </div>
+      </div>
+    </div>
+    <h1 class="title">Back-End Tools</h1>
+    <div class="section">
+      <div class="tech">
+        <img class="logo" src="assets/rails_icon.png" alt="Rails">
+        <div>
+          <h2 class="text title-card">Ruby on Rails</h2>
+          <p class="text age">Release: 2005</p>
+          <p class="text story">Ruby on Rails, or Rails, is a server-side web application framework written in Ruby. Rails is an MVC framework, providing default structures for a database, a web service, and web pages.</p>
+        </div>
+      </div>
+      <div class="tech">
+        <img class="logo" src="assets/express_icon.svg" alt="Express">
+        <div>
+          <h2 class="text title-card">NodeJS</h2>
```

Comment:
you can define a text class and make the classes title age and story extend from that